lib/reversetunnel/transport.go

```diff
@@ -51,6 +62,26 @@ type TunnelAuthDialer struct {
 	Log logrus.FieldLogger
 }
 
+func (c *TunnelAuthDialerConfig) CheckAndSetDefaults() error {
+	if c.ProxyAddr == "" {
```

Comment:
ProxyAddr can't be legitimately empty, right? Just asking cause before this change, I don't think there was a check for it being not empty, so I'm just wondering if that was an omission (most likely) or a valid use-case.

Reply:
Yes, the ProxyAddr should be alway present.
 The Addr is returned by  ` process.findReverseTunnel` function and passed either to directDial or proxyDial Dial call where is used so empty value doesn't make any sense.